### PR TITLE
ETCM-8504 add check around registrations on rpc test

### PR DESCRIPTION
**Pull request by ladamesny**
# Description

Add your description here, if it fixes a particular issue please provide a
[link](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword=)
to the issue.

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages.
- [ ] New tests are added if needed and existing tests are updated.
- [ ] Relevant logging and metrics added
- [ ] CI passes. See note on CI.
- [ ] Any changes are noted in the `changelog.md` for affected crate
- [ ] Self-reviewed the diff

# Note on CI
If your PR is from a fork, the necessary CI jobs won't trigger automatically for security reasons.
You will need to get someone with write privileges.  Please contact IOG Partner Chains developers to do this
for you.

### CODE REVIEW

**Comment by ladamesny** [comment]
Replaces this PR in Sidechains-tests: https://github.com/input-output-hk/sidechains-tests/pull/307

File: E2E-tests/tests/test_rpc.py [hunk 1]
```diff
@@ -120,8 +120,10 @@ def test_get_registrations(self, api: BlockchainApi):
         registrations = api.get_registrations(mc_epoch, mainchain_key)
 
         assert isinstance(registrations, list)
-        if registrations:
+        if len(registrations) > 0:
```
Comment (rsporny) [inline E2E-tests/tests/test_rpc.py:123]: if registration list is empty test will pass, is this expected?

Comment (ladamesny) [reply E2E-tests/tests/test_rpc.py:123]: no, really?

len([]) == 0

0 > 0 should be false.

I should check for an empty list?

Comment (ladamesny) [reply E2E-tests/tests/test_rpc.py:123]: I'll do something like `if registrations is not empty`

**Review by rsporny — APPROVED** [review]

**Pull request merged**